core/controlplane/config/templates/cluster.yaml

```diff
@@ -387,6 +387,16 @@ worker:
 #      # Documentation: http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/dedicated-instance.html
 #      tenancy: default
 #
+#      # (Experimental) GPU Driver installation support
+#      # Currently, only Nvidia driver is supported.
+#      # This setting takes effect only when node's instance family is p2 of g2.
+#      # Otherwise, installation will be skipped even if enabled.
+#      # Make sure to choose 'docker' as container runtime when enabled this feature.
+#      gpu:
+#        nvidia:
+#          enabled: true
```

Comment:
I guess an installed driver can be unusable once Container Linux is updated afterwards, due to an updated kernel.
// Do you think so too?
Then, I believe we'd better document it - maybe something like:

> Ensure that automatic Container Linux is disabled(it is disabled by default btw). Otherwise the installed driver may stop working when an OS update resulted in an updated kernel

would work.